### PR TITLE
Reviewed features section

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

File: nservicebus/pipeline/features.md [hunk 1]
```diff
@@ -16,21 +16,21 @@ related:
...
+While NServiceBus provides interfaces to plug in code at certain steps in the life-cycle, Features offer a more complete approach to write and distribute custom extensions.
 
 Features allow:
 
- * Configure required dependencies for the Feature.
- * Enable or disable a Feature via configuration or based on conditions and dependencies.
- * Get full access to the pipeline, settings and the container.
+ * Configure required dependencies.
+ * Enabling or disabling via configuration or based on conditions and dependencies.
+ * Full access to the pipeline, settings and the container.
```
Comment (weralabaj) [inline nservicebus/pipeline/features.md]: You're not consistent in the list, choose one option and stick to it :)
e.g. configuring, enabling, accessing, etc.

File: nservicebus/pipeline/features.md [hunk 2]
```diff
@@ -16,21 +16,21 @@ related:
  - samples/startup-shutdown-sequence
 ---
 
-While NServiceBus provides some simple interfaces to plug in the code at certain steps in the life-cycle, Features offer a more complete approach to write and distribute custom extensions.
+While NServiceBus provides interfaces to plug in code at certain steps in the life-cycle, Features offer a more complete approach to write and distribute custom extensions.
```
Comment (weralabaj) [inline nservicebus/pipeline/features.md]: > distribute

Not sure what distributing extensions means in this context? e.g. as a nuget package/library? should we spell it out?

Comment (weralabaj) [inline nservicebus/pipeline/features.md]: > distribute

Not sure what distributing extensions means in this context? e.g. as a nuget package/library? should we spell it out?

File: nservicebus/pipeline/features.md (hunk 1 above)
Comment (weralabaj) [inline nservicebus/pipeline/features.md]: You're not consistent in the list, choose one option and stick to it :)
e.g. configuring, enabling, accessing, etc.

File: nservicebus/pipeline/features.md (hunk 2 above)
Comment (danielmarbach) [inline nservicebus/pipeline/features.md]: @weralabaj how about we just write?

> to write custom extensions.

**Pull request merged**